presto-verifier/src/test/java/com/facebook/presto/verifier/retry/TestRetryDriver.java

```diff
@@ -108,7 +109,7 @@ public void testNonRetryableFailure()
     @Test(timeOut = 5000)
     public void testBackoffTimeCapped()
     {
-        verificationContext = new VerificationContext();
+        VerificationContext verificationContext = new VerificationContext();
```

Comment:
@caithagoras I understand `@Test(singleThreaded = true)` change, but why is this change?

Reply:
Actually, I intend to delete the line because the field `verificationContext` is already initialized and can be used directly.